steps/bitrise-step-appledoc/1.0.1/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Appledoc
```

Comment:
a steplib PR can contain only one step.yml pls read our step development guideline and do a fresh re-share